src/main/java/com/mapzen/jpostal/ExpanderOptions.java

```diff
@@ -176,10 +176,6 @@ public void setRomanNumerals(boolean romanNumerals) {
     }
 
     public static class Builder {
-        static {
-            System.loadLibrary("jpostal"); // Load native library at runtime
-        }
-
```

Comment:
Class loading a builder class for an options class should be lightweight and not require a native library/shared object load!

The solution in this PR is to give the `AddressExpander` singleton single responsibility over the native library load. When the singleton is created, it both loads the shared object/native library and sets the default options.